python/xorbits/numpy/numpy_adapters/tests/test_numpy_adapters.py

```diff
@@ -192,3 +206,8 @@ def test_docstring():
     assert docstring is not None and docstring.endswith(
         "This docstring was copied from numpy.random."
     )
+
+    docstring = xnp.ndarray.__doc__
```

Comment:
Could you also test a class member of `ndarray`? Since the class docstring of `xorbits.numpy.ndarray` has already been taken care before this PR.